src/HelpScoutDocs/DocsApiClient.php

```diff
@@ -25,21 +25,31 @@
  * @package HelpScoutDocs
  */
 class DocsApiClient {
-    
+
     const USER_AGENT = 'Help Scout API/Php Client v1';
     const API_URL = 'https://docsapi.helpscout.net/v1/';
-    
+
     private $userAgent = false;
     private $apiKey    = false;
     private $isDebug   = false;
     private $debugDir  = false;
     private $httpClient;
+    private $lastRawResponse = null;
```

Comment:
Since Guzzle is used the response isn't technically raw (not a string). It's rather Response value object.
So it would be better to rename it to just `lastResponse`.